pkg/volume/plugins.go

```diff
@@ -98,6 +97,14 @@ type DeletableVolumePlugin interface {
 	NewDeleter(spec *Spec) (Deleter, error)
 }
 
+// CreatableVolumePlugin is an extended interface of VolumePlugin and is used to create persistent volumes for the cluster.
+type CreatableVolumePlugin interface {
+	VolumePlugin
+	// NewCreater creates a new volume.Creater which knows how to create PersistentVolumes in accordance with
```

Comment:
I suppose that can be true for a Creater, though I think we should probably encourage the limitation to the PV framework.  A caller could theoretically use Creater to create a volume and stick it on a pod.  I think that pattern would be to create a claim and stick that on a pod.

Recycler and Deleter only work on PVs, so I think the specific comment on those two is appropriate.